src/uct/ib/mlx5/ib_mlx5_log.c

```diff
@@ -469,7 +469,7 @@ void __uct_ib_mlx5_log_rx(const char *file, int line, const char *function,
     char buf[256] = {0};
     size_t length;
 
-    length = ntohl(cqe->byte_cnt);
+    length = ntohl(cqe->byte_cnt) & UCT_IB_MLX5_MP_RQ_BYTE_CNT_MASK;
```

Comment:
this will not work if big segment size configured

Reply:
is it possible to configure large segment size? seems rc poll-rx code today already does same & operation..

Reply:
it is, if MP XRQ is disabled

Reply:
so seems poll rx is also broken? https://github.com/openucx/ucx/blob/f9a69a671d1eafd20dce83b231e4c543a38d02bf/src/uct/ib/rc/accel/rc_mlx5.inl#L1389

Reply:
ahh, sorry - forgot we updated it. Yes it is not possible